.eslintrc.js

```diff
@@ -75,6 +75,16 @@ module.exports = {
     '@typescript-eslint/ban-ts-comment': ['warn'],
     '@typescript-eslint/no-var-requires': ['warn'],
   },
+  overrides: [
+    {
+      files: ['*.js', '*.vue'],
+      extends: ['plugin:jsdoc/recommended'],
+      rules: {
+        'jsdoc/check-tag-names': ['warn', { definedTags: ['values'] }],
```

Comment:
What is the `values` tag?

Reply:
Nice, for anyone else curious about that, documentation is here: https://vue-styleguidist.github.io/docs/Documenting.html#available-tags

Should we add a link to that in the rule configuration so future Openers won't have the same question?